docs/visual-testing/integrations/cypress.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_label: Cypress
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cypress Integration
+
+## Introduction
+
+This guide requires an existing Cypress project.<br />
+You can alternatively take a look to our [example repository](#example).
+
+Sauce Visual provides an integration with [Cypress](https://cypress.io) through a plugin that you can add to any existing Cypress project.
+
+Sauce Labs Visual introduce a new Cypress command:
+- `cy.visualCheck()`: Takes a screenshot and send it to Sauce Labs Visual for comparison.
+
+## Quickstart
+
+### Step 1: Add Sauce Visual for Cypress dependency
+
+- Install the Sauce Visual for Cypress plugin in your current project.
```

Comment:
Since we rely on Cypress, that means that Cypress won't be supported where our plugin can. 
Eventually we can mention which version is supported (currently at least 12+), not sure about node version.

Reply:
I've added more information on how to use Cypress on Sauce. That will guide the customers to a doc with what is supported